### PR TITLE
edits to deal with Issue 103

### LINKED ISSUE #103: Results/catalog section
I was hoping to revamp the Results/catalog section but have run out of time. Here's what I was thinking:

- Change the name to "Species catalog" (having stdpopsim in section headers looks ugly IMO)
- Change the first paragraph to be a summary of what the catalog *is* not what it *contains*. It's a systematic organisation of key quantitative data needed to simulate a given species. 
- The next paras can then talk about table 1 and what's *in* the catalog. It would be nice to introduce the IDs for the models here, and then use the IDs subsequently (see #42)
-  Perhaps finish up with a para where we emphasise the point that we've put a lot of effort into making sure that stuff in the catalog will have long-term stable IDs. This is super important for reproducibility. Good note to finish the section on.

### PR DESCRIPTION
**Pull request by andrewkern**
closes https://github.com/popsim-consortium/manuscript/issues/103

### CODE REVIEW

**Review by dschride — APPROVED** [review]
Overall looks good. Just some nitpicks. 1) Shouldn't we capitalize Eurasian? 2) Hyphens! Several cases where "two-population model", "three-population model" "two epoch model", whatever are not hyphenated. Sorry for missing these earlier!

**Comment by andrewkern** [comment]
i can make these edits before merge. are there some particular hyphens you would like? can you mark them in the text for me?

File: popSimManu.tex [hunk 1]
```diff
@@ -344,17 +349,19 @@ \subsection*{The \stdpopsim Catalog}
...
 At the time of writing, more models of human history have been implemented than for
-any other organism. These models include: a simplified version of the \cite{tennessen2012evolution}
+any other organism (see Table \ref{tab:catalog}). 
+These models include: a simplified version of the \cite{tennessen2012evolution}
 model with only the African population specified (expansion from the ancestral
-population and recent growth), the three population model of \cite{gutenkunst2009inferring}
+population and recent growth; Africa\_1T12), the three population model of \cite{gutenkunst2009inferring}
 which specifies the out-of-Africa bottleneck as well as the subsequent divergence of
-the European and Asian populations, the \cite{tennessen2012evolution} two population variant of the
+the European and Asian populations (OutOfAfrica\_3G09), the \cite{tennessen2012evolution} two population variant of the
```
Comment (dschride) [inline popSimManu.tex]: "two-population variant"

File: popSimManu.tex [hunk 2]
```diff
@@ -363,20 +370,23 @@ \subsection*{The \stdpopsim Catalog}
...
 model for \emph{P. abelii} and \emph{E. coli}.
 For \emph{D. melanogaster} we have implemented the three-epoch model estimated by \cite{sheehan2016deep} from
-an African sample, as well as the out-of-Africa divergence
+an African sample (African3Epoch\_1S16), as well as the out-of-Africa divergence
 and associated bottleneck model of \cite{li2006inferring} which jointly models African
-and European populations. For \emph{A. thaliana}, we implemented the
+and European populations (OutOfAfrica\_2L06). For \emph{A. thaliana}, we implemented the
 model in \cite{durvasula2017african} inferred using \MSMC. This model includes
 a continuous change in population size over time, rather than pre-specified epochs of different
-population sizes. We have also implemented a two epoch and a three epoch model estimated from African
```
Comment (dschride) [inline popSimManu.tex]: "two-epoch and a three-epoch model"

File: popSimManu.tex [hunk 3]
```diff
@@ -363,20 +370,23 @@ \subsection*{The \stdpopsim Catalog}
...
-samples of \emph{A. thaliana} in \cite{huber2018gene}. 
+population sizes (SouthMiddleAtlas\_1D17). We have also implemented a two epoch and a three epoch model estimated from African
```
Comment (dschride) [inline popSimManu.tex]: "two-epoch and a three-epoch model"

File: popSimManu.tex [hunk 4]
```diff
@@ -344,17 +349,19 @@ \subsection*{The \stdpopsim Catalog}
 \end{table}
 
 At the time of writing, more models of human history have been implemented than for
-any other organism. These models include: a simplified version of the \cite{tennessen2012evolution}
+any other organism (see Table \ref{tab:catalog}). 
+These models include: a simplified version of the \cite{tennessen2012evolution}
 model with only the African population specified (expansion from the ancestral
-population and recent growth), the three population model of \cite{gutenkunst2009inferring}
+population and recent growth; Africa\_1T12), the three population model of \cite{gutenkunst2009inferring}
```
Comment (dschride) [inline popSimManu.tex]: "three-population model"

**Comment by dschride** [comment]
Okay those comments mark the ones I noticed.

**Comment by andrewkern** [comment]
>  1) Shouldn't we capitalize Eurasian? 
fixed

File: popSimManu.tex (hunk 3 above)
Comment (andrewkern) [reply popSimManu.tex]: fixed

**Comment by andrewkern** [comment]
@dschride how well am i dealing with @jeromekelleher's comments here https://github.com/popsim-consortium/manuscript/issues/103 ? if sufficiently i'll merge

**Comment by dschride** [comment]
All looks good for #103 but #42 may still need work (i.e. we introduce model labels here but may wish to use them below in the MS).

**Pull request merged**